src/services/GetServerStatus.ts

```diff
@@ -5,11 +5,11 @@ import {
 } from 'openapi2typescript';
 import { useQuery } from 'react-fetching-library';
 
-import { Operations } from '../generated/OpenapiPrivate';
+import { Operations } from '../generated/OpenapiNotifications';
 import { toServer } from '../types/adapters/ServerAdapter';
```

Comment:
We no longer need the `toServer`
```suggestion
import { toNotificationEvent } from '../types/adapters/NotificationEventAdapter';
```